packages/@react-spectrum/form/docs/Form.mdx

```diff
@@ -0,0 +1,224 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/form';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+ import {Checkbox} from '@react-spectrum/checkbox';
+ import {Form} from '@react-spectrum/form';
+ import {Radio, RadioGroup} from '@react-spectrum/radio';
+ import {TextField} from '@react-spectrum/textfield';
+ ```
+
+# Form
+
+<p>{docs.exports.Form.description}</p>
+
+<HeaderInfo
+ packageData={packageData}
+ componentNames={['Form']} />
+
+## Example
+
+```tsx example
+<h3 id="label-1">Login</h3>
+<Form width={400} aria-labelledby="label-1">
+  <TextField label="Email" placeholder="abc@adobe.com" />
+  <TextField label="Password" placeholder="1234" />
+  <Checkbox>Remember me</Checkbox>
+</Form>
+```
+
+## Content
+
+Form accepts one or more children to render as its form elements. Label alignment, positioning, and other [properties](#visual-options)
+are set on the Form itself and propagated to its children to generate a cohesive design. To override these top level Form props on individual children, 
+set them directly on the child instead.
+
+```tsx example
+<h3 id="label-2">Sign Up</h3>
+<Form width={400} aria-labelledby="label-2" isRequired necessityIndicator="label">
+  <TextField label="Name" placeholder="Bob Doe" />
+  <TextField label="Email" placeholder="abc@adobe.com" />
+  <TextField label="Address" placeholder="123 Any Street" isRequired={false} />
+</Form>
+```
+
+### Accessibility
+
+As per [W3 Form guidelines](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/form.html), the controls provided to the form should 
+use native host semantics (`button`, `input`, `select`, `textarea`) whenever possible.
+
+## Labeling
+
+### Accessibility
+
+If a label is provided to the Form by adding either the `aria-label` or `aria-labelledby` prop, 
+the element will be identified to assistive technology as a form landmark region. 
```

Comment:
From v2 docs, but w3 guidelines (https://www.w3.org/TR/wai-aria-practices/examples/landmarks/form.html) state that "A label for the form landmark should be identified using aria-labelledby to visible heading element (e.g. an h1-h6 element)". Not sure which to go with

Reply:
Isn't the form landmark region determined by using the form element or a role="form" and not related to the labeling?

Reply:
Not too sure myself, the above sentence was pilfered from the v2 docs. @majornista could you shed some light here? Is the above sentence something specific to v2 or should I change it to something like "A label for the form landmark should be identified using aria-labelledby to visible heading element (e.g. an h1-h6 element)"?

Reply:
A form element without a label will not be identified as a landmark region to assistive technology. This is similar to the behavior of `<section>` or elements with `role="region"`, without a label, the element is treated by assistive technology as presentational, but with a label, the element is treated as a landmark region, to which assistive technology can navigate.

Reply:
Chrome extension worth installing: http://matatk.agrip.org.uk/landmarks/

Reply:
Cool, thanks for the info. I'll add the "without a label" bit to this section